python/revng/api/_capi.py

```diff
@@ -79,8 +79,7 @@ def __setattr__(self, name, value):
             raise NotImplementedError()
 
 
-# We assume this file is in <root>/lib/python/revng/api
-ROOT = (Path(__file__) / "../../../../..").resolve()
```

Comment:
There's also this:
```
python/revng/cli/support.py:root_path = os.path.join(script_path, "..", "..", "..", "..")
```
Can we unify?

Reply:
Ah, I grepped `../..` and missed it, fixing